runtime/development/src/lib.rs

```diff
@@ -1814,14 +1814,16 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	SchedulerMigrationV3,
+	UpgradeDev1019,
 >;
 
+type UpgradeDev1019 = SchedulerMigrationV4;
+
 // Migration for scheduler pallet to move from a plain Call to a CallOrHash.
-pub struct SchedulerMigrationV3;
-impl frame_support::traits::OnRuntimeUpgrade for SchedulerMigrationV3 {
+pub struct SchedulerMigrationV4;
+impl frame_support::traits::OnRuntimeUpgrade for SchedulerMigrationV4 {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		Scheduler::migrate_v2_to_v4()
+		Scheduler::migrate_v3_to_v4()
```

Comment:
Develop on `1018` has `Scheduler.PalletVersion(3)`. Thus, we need to migrate to V4.